README.md

```diff
@@ -2,7 +2,7 @@
 
 # Coming Soon! Digital Identity Guidelines: Public Comment Period  
 
-A huge thank you to all that contributed to the public preview of draft Special Publication 800-63-3. We are suspending the collection of issues and comments for the public preview to get ready for our next phase. As we mentioned when we launched this May, we will be transitioning into a more formal process of gathering feedback from the stakeholder community in a manner that aligns with how agencies and NIST expect to participate in reviewing SP 800-63-3.  This next part of our process is quickly coming.  Until then, the public preview pages will remain viewable.  You can find more details about next steps at [our blog site](http://nstic.blogs.govdelivery.com/).
+A huge thank you to all that contributed to the public preview of draft Special Publication 800-63-3. We are suspending the collection of issues and comments for the public preview to get ready for our next phase. As we mentioned when we launched this May, we will be transitioning into a more formal process of gathering feedback from the stakeholder community in a manner that aligns with how agencies and NIST expect to participate in reviewing SP 800-63-3.  This next part of our process is quickly coming.  Until then, the public preview pages will remain viewable.  You can find more details about next steps at [our blog site](http://trustedidentities.blogs.govdelivery.com/2016/09/20/many-thanks-for-a-successful-800-63-3-public-preview/).
```

Comment:
It's not possible to test this, because (as someone else pointed out) DNS is not working for trustedidentities.blogs.govdelivery.com. Has this been reported?